src/karts/ghost_kart.cpp

```diff
@@ -130,9 +138,60 @@ void GhostKart::update(int ticks)
         m_all_physic_info[idx].m_steer, m_all_physic_info[idx].m_speed,
         /*lean*/0.0f, idx);
 
+    // Attachment management
+    // FIXME : don't handle if an attachment is renewed
+    //         e.g., a bubble shield is used again with another active
+    //         potentially reaching arbitrary high active time
+
+    Attachment::AttachmentType attach_type = ReplayRecorder::codeToEnumAttach(m_all_bonus_info[idx].m_attachment); 
+    int attach_ticks = 0;
+    // FIXME The exact value depends on kart properties. Should it be written in the replay file ?
```

Comment:
I think this is only a visual effect so I don't think accuracy is all that important, but maybe add a little comment explaining those are approximate durations used for visualisation only?

Reply:
Also that line looks way too long - ~80 characters maximum!